h/templates/app.html.jinja2

```diff
@@ -13,10 +13,6 @@
               href="{{ link.href }}"/>
       {% endfor %}
     {% endif %}
-
-    <!-- Prefetch non-authenticated API routes which client will need. !-->
-    <link rel="preload" as="fetch" href="{{ request.route_url("api.index") }}" crossorigin="anonymous">
-    <link rel="preload" as="fetch" href="{{ request.route_url("api.links") }}" crossorigin="anonymous">
```

Comment:
As an ultra performance optimisation, we could add these links with attribute `as="prerender"` from our pages in `web.hypothes.is`:  https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/prerender

That way they will be already in the browser cache.

Given (1) the lack of universal browser support of the `as="prerender"`, and that (2) the client could be rendered from other pages not just after `web.hypothes.is`, I think this is not needed.

Reply:
I didn't know about `<link as="prerender">`, thanks for sharing :) It is interesting that this is not implemented in either Firefox or Safari despite having existed for a very long time in Chrome. It would be interesting to research why that is.